### ISSUE TITLE
Input Not Cleared on Submission

### ISSUE THREAD

**Issue by adsuth** [jrbsu/wikipedia-game#15]
The input box is not cleared when submitting an answer by moving to the correct option with arrow keys and pressing enter
(this one is harder to demonstrate)
![image](https://user-images.githubusercontent.com/74009107/213156636-d532e489-0049-4661-b04f-62769af7f839.png)

The input box suggestion list persists for the last input when you guess an article **without** the suggestion list
![image](https://user-images.githubusercontent.com/74009107/213156206-58d6f5e9-dd1f-47f3-8921-e7de634882d9.png)

Probably just a case of the `#guess` input not having its value set to `""` on each of the difierent submission methods. Just making the issue for the sake of completeness

**Comment by jrbsu** [comment]
Yeah I've noticed this too, very annoying. It's because we're using JqueryUI's `.autocomplete` function which is super helpful _other than_ this bug. There's probably a way to do it. I did try with `return false;` but that introduced a few other problems with the input that weren't worth it.
